workflows/chatbot/inference/generate.py

```diff
@@ -330,6 +336,7 @@ def load_model(
     use_cache=True,
     peft_path=None,
     use_deepspeed=False,
+    hg_access_token=None,
```

Comment:
this seems test specific. Can we move into test script? e.g., workflows/chatbot/test/...

Reply:
> this seems test specific. Can we move into test script? e.g., workflows/chatbot/test/...

for now, it's mandatory to use authorized token to access llama2.

Reply:
ok. change to hf_access_token then.

Reply:
> ok. change to hf_access_token then.

sounds good.

Reply:
renamed to hf_access_token, please check.